zep_python/document/collections.py

```diff
@@ -678,7 +678,7 @@ def search(
         """
 
         results, _ = self.search_return_query_vector(
-            text=text,
+            text=str(text),
```

Comment:
why are you naively converting a possible dict to a string?

Reply:
Rolled back